### PR TITLE
MAGETWO-59112 Backport 2.1.x 

### PR DESCRIPTION
**Pull request by Ctucker9233**
Backport MAGETWO-59112 to 2.1.x

### Description
Backport MAGETWO-59112 to 2.1.x

### Fixed Issues (if relevant)
Invoice Address formating Issue

### Manual testing scenarios

1. ...
2. ...

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=13896) <br/>All committers have signed the CLA.

**Comment by Ctucker9233** [comment]
#6771 This is a backport to fix this issue

**Comment by Ctucker9233** [comment]
Travis build failed and I'm not sure why. Can someone take a look, please?

**Comment by magento-engcom-team** [comment]
@Ctucker9233 thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/AddToCartCrossSellTest.xml [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../../vendor/magento/mtf/etc/variations.xsd">
     <testCase name="Magento\Catalog\Test\TestCase\Product\AddToCartCrossSellTest" summary="Promote Products as Cross-Sells" ticketId="MAGETWO-12390">
         <variation name="AddToCartCrossSellTestVariation1" method="test">
-            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test</data>
+            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test, stable:no</data>
```
Comment (sidolov) [inline dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/AddToCartCrossSellTest.xml]: Please, remove 'stable: no' tag and try to run test with backported fix.

File: dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/NavigateRelatedProductsTest.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../../vendor/magento/mtf/etc/variations.xsd">
     <testCase name="Magento\Catalog\Test\TestCase\Product\NavigateRelatedProductsTest" summary="Promote Products as Related" ticketId="MAGETWO-12392">
         <variation name="NavigateRelatedProductsTestVariation1" method="test">
-            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test</data>
+            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test, stable:no</data>
```
Comment (sidolov) [inline dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/NavigateRelatedProductsTest.xml]: Please, remove 'stable: no' tag and try to run test with backported fix.

File: dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/NavigateUpSellProductsTest.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../../../../../vendor/magento/mtf/etc/variations.xsd">
     <testCase name="Magento\Catalog\Test\TestCase\Product\NavigateUpSellProductsTest" summary="Promote Products as Up-Sells" ticketId="MAGETWO-12391">
         <variation name="NavigateUpSellProductsTestVariation1" method="test">
-            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test</data>
+            <data name="tag" xsi:type="string">test_type:acceptance_test, test_type:extended_acceptance_test, stable:no</data>
```
Comment (sidolov) [inline dev/tests/functional/tests/app/Magento/Catalog/Test/TestCase/Product/NavigateUpSellProductsTest.xml]: Please, remove 'stable: no' tag and try to run test with backported fix.

File: dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php [hunk 4]
```diff
@@ -96,6 +96,7 @@ public function testFormPostInitNew($productData, $categoryData, $action, $reque
      */
     public function testGetEntityStores($productData, $categoryData, $expectedStores)
     {
+        $this->markTestSkipped('Skipped until MAGETWO-63018');
```
Comment (sidolov) [inline dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php]: Please, unskip test and try to run it.

File: dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php [hunk 5]
```diff
@@ -121,6 +122,7 @@ public function testGetEntityStores($productData, $categoryData, $expectedStores
      */
     public function testGetEntityStoresProductStoresException()
     {
+        $this->markTestSkipped('Skipped until MAGETWO-63018');
```
Comment (sidolov) [inline dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php]: Please, unskip test and try to run it.

**Review by sidolov — CHANGES_REQUESTED** [review]

File: dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php (hunk 5 above)
Comment (Ctucker9233) [reply dev/tests/integration/testsuite/Magento/UrlRewrite/Block/Catalog/Edit/FormTest.php]: OK, let me try that.

**Comment by Ctucker9233** [comment]
@sidolov Made changes and waiting to see how the build goes.

**Review by sidolov — APPROVED** [review]

**Pull request merged**